be/src/olap/aggregate_func.h

```diff
@@ -435,7 +436,7 @@ struct AggregateFuncTraits<OLAP_FIELD_AGGREGATION_HLL_UNION, OLAP_FIELD_TYPE_HLL
         slice->data = arena->Allocate(HLL_COLUMN_DEFAULT_LEN);
         slice->size = hll->serialize(slice->data);
 
-        delete hll;
+        hll->~HyperLogLog();
```

Comment:
should change [there](https://github.com/apache/incubator-doris/pull/1778/files#diff-6b1f7d667aa162c48abc8d5fd9b4f6b0R428)